### DISCUSSION TITLE
fs actions using file names instead of absolute file paths

### DISCUSSION THREAD

**Discussion by asror1** [nvim-tree/nvim-tree.lua#2266] [Q&A] [answered]
Hello there! 

If I'm not mistaken it's not currently possible to pass in an option to the **fs** actions to have them use file names instead of absolute paths when sending notifications, something like this... 
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/92228287/c6f06967-7de5-4de3-91ce-7b0d6c830453)

Are there any circumstances that I'm not considering that would make this option not worth adding? Because for me I would much rather prefer a bit of ambiguity in the notifications, as a tradeoff for shorter notification messages, instead of these...

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/92228287/34a1c658-3efd-4ab2-8b27-0da723fbb294)

**Comment by alex-courtis** [comment, marked as answer]
That is of course possible. It could be an option in the `:h nvim-tree.notify` section. Perhaps `notify.name_only` default false, so as not to break existing users.

[Pull requests](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md) are always gratefully appreciated! Please raise a feature request when you raise your PR.

**Reply by asror1** [reply]
Sounds good, I'm on it 👍
